wordpress-mobile/WordPress-Android
# [Gutenberg] Incorporate links-UI work

This PR just updates the hashes to include the work related with the links UI in rich-text feature. There are no changes to wpandroid beyond that.

GB-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/275

To test Aztec (standalone):

* Open or create a post and try out the "italic" feature from the formatting toolbar
* If any changes are made, the html output should have `<em>` tags instead of `<i>` in the touched text

To test Gutenberg:

* Open a gutenberg post and try out adding or editing a link
* There should be UI to create and edit a link

Update release notes:
